src/sections/Home/Partners-home/index.js

```diff
@@ -10,7 +10,6 @@ import "slick-carousel/slick/slick-theme.css";
 
 const settings = {
   initialSlide: 1,
-  lazyLoad: true,
```

Comment:
why lazyload is being a issue here. Lazy loading is kind of important because we don't want to hinder the experience of users if the images taking time to load. So why lazy loading is creating a issue here?

Reply:
@H-A-R-S-H-K @Ashparshp are either of you investigating this?